web/themes/joinup/templates/rdf_entity/rdf-entity--solution--listing-item.html.twig

```diff
@@ -25,10 +25,12 @@
       </div>
     {% endif %}
     <div class="listing__field listing__stats mdl-card__actions">
-      <div class="listing__stat">
-        <span class="listing__text">{{ content.downloads.0 }}</span>
-        <span class="listing__icon icon icon--download"></span>
-      </div>
+      {% if content.downloads  %}
+        <div class="listing__stat">
+          <span class="listing__text">{{ content.downloads.0 }}</span>
```

Comment:
We don't need the `.0` suffix here. Let's keep it clean:
```html
<span class="listing__text">{{ content.downloads }}</span>
```

Reply:
Removed!

Reply:
Looking good now!